File: Arrays/Left_Rotate_Array_By_one.cpp
#include <iostream>
using namespace std;

void lrotate(int arr[] , int n)
{
	int temp  = arr[0];
	for(int i = 1; i < n ; i ++)
	{
		arr[i - 1] = arr[i];
	}
	arr[n - 1] = temp;
}

int main()
{
	int arr[] ={10, 20, 230, 40};
	int n = sizeof(arr) / sizeof(arr[0]);
	lrotate(arr , n);
	for(int i =0 ; i < n ; i ++)
	{
		cout<<"Executed  and the result is  = " << arr[i]<<" ";
	}
}

File: Arrays/Left_Rotate_Array_By_one.java
public class Left_Rotate_Array_By_one {

	static void lRotate(int arr[] , int n)
	{
		int temp = arr[0];
		for(int i = 1; i < n ; i ++)
		{
			arr[i - 1] = arr[i];
		}
		arr[n  -  1] = temp; 
	}

    public static void main(String[] args) {
    	int arr[] = {1, 2, 3, 4, 5};
    	int n = arr.length;
    	lRotate(arr , n );
    	for(int i =0 ; i < n ; i ++)
    	{
    		System.out.print(arr[i]+" ");
    	}
      }
}